packages/params-middleware/README.md

```diff
@@ -0,0 +1,26 @@
+# Params middleware
+
```

Comment:
Usage section is missing? An example explaining what this is about would be helpful.